source/includes/example-connect-onprem-atlas.rst

```diff
@@ -0,0 +1,50 @@
+Gather Connection Information
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The source cluster, ``cluster0``, is hosted on the following servers
+and ports:
+
+- clusterOne01.fancyCorp.com:20020
+- clusterOne02.fancyCorp.com:20020
+- clusterOne03.fancyCorp.com:20020
+
+The destination cluster, ``cluster1``, is hosted on the following
+servers and ports:
+
+- clusterTwo01.mongodb.net:27017
+- clusterTwo02.mongodb.net:27017
+- clusterTwo03.mongodb.net:27017
+
+There is an administrative user, ``clusterAdmin`` configured on each
+cluster with password, ``superSecret``.
+
+Connect the Source and Destination Clusters with ``mongosync``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Use the connection information you gathered to create the connection
+strings for ``cluster0`` and ``cluster1``:
+
+.. code-block:: shell
+
+   cluster0:
+   mongodb://clusterAdmin:superSecret@clusterOne01.fancyCorp.com:20020,clusterOne02.fancyCorp.com:20020,clusterOne03.fancyCorp.com:20020
+   cluster1:
+   mongodb+srv://clusterAdmin:superSecret@clusterTwo01.mongodb.net:27017,clusterTwo02.mongodb.net:27017,clusterTwo03.mongodb.net:27017
```

Comment:
The SRV hostname is a little different, something like:

```
mongodb+srv://clusterAdmin:superSecret@clusterTwo.abc12.mongodb.net
```

Reply:
Blanked on the fact that srv doesn't have hosts/ports! Sorry bout that — updated!